docs/source/developers/guide/step_by_step/building.rst

```diff
@@ -150,5 +150,13 @@ interested to find more information.
 
          - :ref:`build_pyarrow_win`
 
-   .. tab:: Building the R package
 
+   .. tab:: Building the R package
+      
+	  When working on code in the R package, you only need to build the Arrow C++ 
+	  library (often referred to in the R documentation as 'libarrow') from source 
+	  if you wish to make changes to the code there; otherwise you can install a 
+	  binary version of the C++ libary and then need to link it up with the R package.
```

Comment:
I think this still doesn't work - installing `libarrow` from homebrew (on Mac) and having only the R package as `source`.

Reply:
Great point, will update